test/cli-commands/ganache/ganache.js

```diff
@@ -0,0 +1,130 @@
+const assert = require('chai').assert;
+const runCmdHandler = require('../utils/spawn-child-process').runCmdHandler;
+const killProcessByPID = require('../utils/spawn-child-process').killProcessByPID;
+const getGanacheProcessState = require('../utils/child-process-states').getState;
+const timeout = require('../utils/timeout').timeout;
+const hook_stream = require('../utils/hookup-standard-output').hook_stream;
+
+const ganache = require('ganache-cli');
+const setup = require('./setup.json');
+const ganacheRun = require('../../../cli-commands/ganache/ganache').run;
+const listenCallback = require('../../../cli-commands/ganache/ganache').listenCallback;
+
+const RUNNING_GANACHE_TIMEOUT = 10000;
+const SECOND_TIMEOUT = 1000;
+
+const DEFAULT_PORT = '8545';
+const SPECIFIC_PORT = '8123';
+const GANACHE_CLI_SERVER_PORT = '8130';
+
+const ADDRESS_START_INDEX = 13;
+const ADDRESS_LENGTH = 42;
+
+const PRIVATE_KEY_START_INDEX = 69;
+const PRIVATE_KEY_LENGTH = 66;
+
+const FIRST_ACCOUNT_ADDRESS = '0xd9995bae12fee327256ffec1e3184d492bd94c31';
+const FIRST_PRIVATE_KEY = '0x7ab741b57e8d94dd7e1a29055646bafde7010f38a900f55bbd7647880faa6ee8';
+
+const THIRD_ACCOUNT_ADDRESS = '0x760bf27cd45036a6c486802d30b5d90cffbe31fe';
+const THIRD_PRIVATE_KEY = '0x62ecd49c4ccb41a70ad46532aed63cf815de15864bc415c87d507afd6a5e8da2';
+
+const TENTH_ACCOUNT_ADDRESS = '0x87e0ed760fb316eeb94bd9cf23d1d2be87ace3d8';
+const TENTH_PRIVATE_KEY = '0xfac0bc9325ad342033afe956e83f0bf8f1e863c1c3e956bc75d66961fe4cd186';
+
+let ganacheCommandOutput = '';
+
+describe('Ganache cli command', () => {
+
+	beforeEach(() => {
+		ganacheCommandOutput = '';
+	});
+
+	describe('Run ganache server on specific port', async () => {
+		it('should start ganache server on specific port', async () => {
+			runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8123');
```

Comment:
Why not put the `runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8123');` in the beforeEach block. This way you won't be duplicating much code and use many ports

and add killing of the process in `afterEach` block